client/src/app/site/motions/services/motion-pdf-catalog.service.ts

```diff
@@ -243,8 +243,13 @@ export class MotionPdfCatalogService {
      * @returns {Array<Object>} An array containing the `DocDefinitions` for `pdf-make`.
      */
     private appendSubmittersAndRecommendation(motion: ViewMotion, style: StyleType = StyleType.DEFAULT): Object[] {
-        const recommendation = this.motionRepo.getExtendedRecommendationLabel(motion);
         let submitterList = '';
+        let state = '';
+        if (motion.state.isFinalState) {
+            state = this.translate.instant(motion.state.toString());
```

Comment:
You have to use the extended state string - with optional state extension field.